pylintrc

```diff
@@ -102,6 +102,8 @@ disable=
     consider-using-enumerate,  # pylint 2.1, clean up tests later
     no-else-raise, # python 2.4.0
     import-outside-toplevel, # pylint 2.4.2
+    f-string-without-interpolation,  # pylint 2.5.0, bare f-strings are ok
+    too-many-function-args,  # pylint 2.5.0, should be fixed
```

Comment:
This can be fixed with:
```diff
--- a/ipatests/pytest_ipa/integration/host.py
+++ b/ipatests/pytest_ipa/integration/host.py
@@ -134,7 +134,7 @@ class Host(pytest_multihost.host.Host):
         else:
             cls = Host
 
-        return cls(domain, hostname, role, ip, external_hostname)
+        return cls(domain, hostname, role, ip=ip, external_hostname=external_hostname)
 
     def ldap_connect(self):
         """Return an LDAPClient authenticated to this host as directory manager

```

Reply:
Good idea, fixed!